conf/st2.conf.sample

```diff
@@ -29,7 +29,7 @@ max_page_size = 100
 # True to mask secrets in the API responses
 mask_secrets = True
 # StackStorm API server host
-host = 127.0.0.1
```

Comment:
Yeah we can do that - I don't think it should change / break anything, but will see.